samples/qa-videoplayer/src/views/MainPanel.module.less

```diff
@@ -2,6 +2,6 @@
 
 .applySkins({
 	.selecting {
-		border: 6px solid @sand-accent;
+		border: 6px solid @sand-untypical-color-10;
```

Comment:
🤔 Could we just use the hex value of the color here instead of this?
Or have you seen the usage? I think we could use "selected" prop instead of this styling maybe..?
![image](https://user-images.githubusercontent.com/5037429/142982005-a525eaab-248c-4a36-a6f6-51da2aeae2d5.png)

Reply:
I will fix it with the hex value for now. I'm discussing a topic of the UI of the sample in another ticket with @dongsuwon.